src/vs/workbench/contrib/terminalContrib/clipboard/browser/terminalClipboard.ts

```diff
@@ -9,14 +9,84 @@ import { IDialogService } from '../../../../../platform/dialogs/common/dialogs.j
 import { ServicesAccessor } from '../../../../../platform/instantiation/common/instantiation.js';
 import { TerminalSettingId } from '../../../../../platform/terminal/common/terminal.js';
 
-export async function shouldPasteTerminalText(accessor: ServicesAccessor, text: string, bracketedPasteMode: boolean | undefined): Promise<boolean | { modifiedText: string }> {
+class SmartPasteUtils {
```

Comment:
`namespace smartPaste` is preferable to a class containing only `static`s. However just moving these into a separate file inside `terminalContrib/clipboard` would be best